hack/convert-output-to-tsv.py

```diff
@@ -36,34 +38,64 @@ def remove_prefix(text, prefix):
 
 gcs = GCSFileSystem()
 
+# create a service object to handle all aspects of generating a primary key
+@dataclass
+class PrimaryKeyGenerator:
+    table_name: str
+    pk_name: str = field(init=False)
+    firecloud: bool
+
+    # this will calculate pk_name during init
+    def __post_init__(self):
+        # most tables should have "dog_id" as a key
+        if self.table_name in {"hles_dog", "hles_cancer_condition", "hles_health_condition", "environment", "cslb"}:
+            self.pk_name = 'dog_id'
+        # owner table is linked to hles_dog via "owner_id"
+        elif self.table_name == 'hles_owner':
+            self.pk_name = 'owner_id'
+        else:
+            raise ValueError(f"Unrecognized table: {self.table_name}")
+
+    # this will return the firecloud entity names
```

Comment:
i think this information is conveyed by the method name, so the comment can be nixed